### PR TITLE
Clean up UI around missing multimedia references

### PR DESCRIPTION
**Pull request by orangejenny**
Minor changes, read commit list for details.

Multimedia tab on app settings
screen shot 2019-01-14 at 2 53 10 pm

Multimedia reference checker
screen shot 2019-01-14 at 2 53 24 pm

@esoergel 
fyi @dimagi/product

### CODE REVIEW

**Comment by orangejenny** [comment]
pinging for review - @esoergel or new code buddy @millerdev

File: corehq/apps/app_manager/templates/app_manager/partials/settings/multimedia_ajax.html [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
     <div class="panel-body">
-      <p>{% blocktrans %}View, upload, and download your application's multimedia{% endblocktrans %}</p>
       <p>
-          <a id="open_checker" target="_blank" class="btn btn-primary" href="{% url "hqmedia_references" domain app.get_id %}">
-              <i class="fa fa-check"></i>
-              {% blocktrans %}Multimedia Reference Checker</a> (opens in a new window){% endblocktrans %}
+          <a id="open_checker" target="_blank" href="{% url "hqmedia_references" domain app.get_id %}">
+              {% blocktrans %}View, upload, and download your application's multimedia{% endblocktrans %}
+              <i class="fa fa-external-link"></i>
+          </a>
```
Comment (esoergel) [inline corehq/apps/app_manager/templates/app_manager/partials/settings/multimedia_ajax.html]: Nice copy changes

**Review by esoergel — APPROVED** [review]
All looks good!  Sorry this dropped off my radar

**Comment by esoergel** [comment]
haha, jinx

**Pull request merged**